modules/client/src/addressList/internal/client/methods.ts

```diff
@@ -46,6 +48,10 @@ import { AllowlistVoting__factory } from "@aragon/core-contracts-ethers";
 import { id } from "@ethersproject/hash";
 import { hexZeroPad } from "@ethersproject/bytes";
 import { toUtf8Bytes } from "@ethersproject/strings";
+import {
+  UNAVAILABLE_PROPOSAL_METADATA_LINK,
```

Comment:
The thing that is unavailable is the metadata, not the link IMO
```suggestion
  UNAVAILABLE_PROPOSAL_METADATA,
```